agent/exploits/cve_2023_1389.py

```diff
@@ -61,7 +61,7 @@ def _generate_vulnerability_object(
     ) -> definitions.Vulnerability:
         entry = kb.Entry(
             title=VULNERABILITY_TITLE,
-            risk_rating="POTENTIALLY",
+            risk_rating="CRITICAL",
```

Comment:
Why this is ` CRITICAL` and you change it later to POTENTIALLY?

Reply:
it reports too many FPs, we set it to potentially until it's improved